.azdo/pipelines/app-ci.yaml

```diff
@@ -71,7 +71,9 @@ stages:
   - deployment: DeploySpringbootApp
     displayName: 'Azure Web App' 
     variables:
-      buildjarfilename: $[ stageDependencies.Build.WebApp.outputs['pomvariable.jarfilename'] ]
+    - group: azureSearchOpenAiDemoJavaDev
```

Comment:
I think the previous version which was using a simple variable defined as build step output was good. what advantages you see to use a variable group here?